deployments/gcp-uscentral1b/config/common.yaml

```diff
@@ -90,6 +94,40 @@ pangeo:
           name: custom-templates
           subPath: "pangeo-custom-jupyterhub-templates/extra-assets"
 
+  dask-gateway:
+    gateway:
+      extraConfig:
+        optionHandler: |
+          from dask_gateway_server.options import Options, Integer, Float, String
+          def cluster_options(user):
+             def option_handler(options):
+                 if ":" not in options.image:
+                     raise ValueError("When specifying an image you must also provide a tag")
+                 extra_annotations = {
+                     "hub.jupyter.org/username": user.name,
+                     "prometheus.io/scrape": "true"
```

Comment:
This is the only line we *really* care about. Enables prometheus metric collection for scheduler & worker pods created by dask-gateway.